{"name": "Тест авторизации с неверным логином", "status": "broken", "statusDetails": {"message": "AttributeError: 'Response' object has no attribute 'copy'", "trace": "self = <tests.test_login_user.TestLoginUser object at 0x00000223926CF280>, user_data = <Response [403]>\n\n    @allure.title('Тест авторизации с неверным логином')\n    def test_login_user_incorrect_email_error(self, user_data):\n>       payload = user_data.copy()\nE       AttributeError: 'Response' object has no attribute 'copy'\n\ntests\\test_login_user.py:17: AttributeError"}, "start": 1709681156714, "stop": 1709681156715, "uuid": "408aebb3-1c4a-4317-adda-17afa672b654", "historyId": "27943f6730ab4d740c33daa26147545a", "testCaseId": "27943f6730ab4d740c33daa26147545a", "fullName": "tests.test_login_user.TestLoginUser#test_login_user_incorrect_email_error", "labels": [{"name": "feature", "value": "Проверяем авторизацию пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-AQ71LTK"}, {"name": "thread", "value": "10368-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}